Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.7-alpine
+
+RUN apk add --no-cache --virtual build-dependencies gcc musl-dev
+RUN pip3 install --no-cache-dir homeassistant-cli==0.4.4
```

Comment:
install from source, just use `pip3 install .`

should also add a .dockerignore to avoid unnecessary cruft.

Reply:
yes, i would prefer building from source too. will implement this today or tomorrow morning